pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -55,6 +55,14 @@ self: super: {
   # > https://github.com/roelvandijk/numerals
   numerals = doJailbreak (dontCheck super.numerals);
 
+  # xmlhtml's test suite depends on hspec with an invalid boundry range for
+  # the version we currently track, even though the upper bound is relaxed on
```

Comment:
Why can't we use `doJailbreak`?

Reply:
Yes, I'd prefer test being executed any day.

Reply:
Alright, updated :+1: